scripts/apps/archive/constants.ts

```diff
@@ -114,3 +117,44 @@ export const CANCELED_STATES = KILLED_STATES.concat([ITEM_STATE.SPIKED]);
  * KILLED | RECALLED | UNPUBLISHED | SPIKED | SCHEDULED
  */
 export const READONLY_STATES = CANCELED_STATES.concat([ITEM_STATE.SCHEDULED]);
+
+export const IPTC_XMP_TAGS = {
```

Comment:
Could you adjust this so group names aren't concatinated and don't need to be stripped later. It's not serving any purpose as far as I can see.

use one variable that defines `ipct_to_xmp` and `xmp_to_iptc = invert(ipct_to_xmp)`

use a helper from lodash https://lodash.com/docs/4.17.15#invert

Reply:
Sorry for a late response I was on vacation. Could you add unit test(s) to cover tags existing in both groups?